src/components/CurrentRefinements/CurrentRefinements.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'preact-compat';
+import PropTypes from 'prop-types';
+import isEqual from 'lodash/isEqual';
+import upperFirst from 'lodash/upperFirst';
```

Comment:
this is actually a quite big package, and I think we only use it here. We can rewrite it (later?) to a smaller version that doesn't deal with so many edge cases as the lodash version